runelite-client/src/main/java/net/runelite/client/plugins/config/ConfigPanel.java

```diff
@@ -72,37 +71,17 @@
 	private static final int SPINNER_FIELD_WIDTH = 6;
 
 	private final ConfigManager configManager;
-	private JTextField searchBar;
+	private JTextField searchBar = new JTextField();
+	private Map<String, JPanel> children = new TreeMap<>();
+	private int scrollBarPosition = 0;
 
 	public ConfigPanel(ConfigManager configManager)
 	{
 		super();
 		this.configManager = configManager;
-		populateConfig();
-	}
-
-	@Override
-	public void invalidate()
-	{
-		super.invalidate();
-
-		if (searchBar != null)
-		{
-			searchBar.requestFocusInWindow();
-		}
-	}
-
-	private void populateConfig()
```

Comment:
Most of the construction in this method gets moved to the constructor, and all of the modification to the component tree goes to openConfigList.